plugins/newsdownloader.koplugin/main.lua

```diff
@@ -163,89 +247,188 @@ function NewsDownloader:loadConfigAndProcessFeeds()
         local limit = feed.limit
         local download_full_article = feed.download_full_article == nil or feed.download_full_article
         local include_images = not never_download_images and feed.include_images
         local enable_filter = feed.enable_filter or feed.enable_filter == nil
         local filter_element = feed.filter_element or feed.filter_element == nil
+        -- Check if the two required attributes are set
         if url and limit then
-            local feed_message = T(_("Processing %1/%2:\n%3"), idx, total_feed_entries, BD.url(url))
+            feed_message = T(_("Processing %1/%2:\n%3"), idx, total_feed_entries, BD.url(url))
             UI:info(feed_message)
-            NewsDownloader:processFeedSource(url, tonumber(limit), unsupported_feeds_urls, download_full_article, include_images, feed_message, enable_filter, filter_element)
+            -- Process, i.e.: "Download"
+            self:processFeedSource(
+                url,
+                tonumber(limit),
+                unsupported_feeds_urls,
+                download_full_article,
+                include_images,
+                feed_message,
+                enable_filter,
+                filter_element)
         else
-            logger.warn('NewsDownloader: invalid feed config entry', feed)
+            logger.warn('NewsDownloader: invalid feed config entry.', feed)
         end
     end
 
     if #unsupported_feeds_urls <= 0 then
-        UI:info("Downloading news finished.")
+        -- When no errors are present, we get a happy message.
+        feed_message = "Downloading news finished. "
     else
+        -- When some errors are present, we get a sour message.
         local unsupported_urls = ""
-        for k,url in pairs(unsupported_feeds_urls) do
-            unsupported_urls = unsupported_urls .. url
-            if k ~= #unsupported_feeds_urls then
+        for key, value in pairs(unsupported_feeds_urls) do
+            -- Create the error message
+            unsupported_urls = unsupported_urls .. " " .. value[1] .. " " .. value[2]
+            -- Not sure what this does
+            if key ~= #unsupported_feeds_urls then
                 unsupported_urls = BD.url(unsupported_urls) .. ", "
             end
         end
-        UI:info(T(_("Downloading news finished. Could not process some feeds. Unsupported format in: %1"), unsupported_urls))
+        -- Tell the user there were problems
+        feed_message = "Downloading news finished with errors. "
+        go_to_config_file = UI:confirm(
+            T(
+                [[
+Could not process some feeds.
+Unsupported format in: %1. Please
+review your feed configuration file.]], unsupported_urls),
+            "Continue",
+            ""
+        )
+    end
+    -- Clear the info widgets before displaying the confirm box
+    UI:clear()
+    -- Ask the user if they want to see their downloads
+    feed_message = feed_message .. "Go to downloaders folder?"
```

Comment:
All such strings should be _("string") for translation.

```suggestion
    feed_message = feed_message .. _("Go to download folder?")
```